### ISSUE TITLE
Remove toasters and add an activity page

### ISSUE THREAD

**Issue by icrc-fdeniger** [icrc/openmrs-android-fhir#68]
The application currently uses toast notifications to inform users about ongoing synchronization tasks and issues. These notifications should be replaced by an "Activity Page" that provides detailed synchronization information for each sync session.

For each synchronization session, a block of information will be displayed, including:

- Synchronization start date and time (used to identify each session).
- Number of patients pulled from the server out of the total patients that need to be synchronized.
- Number of patients pushed to the server out of the total locally modified or newly created patients.
- Synchronization completion time.

For each block, a status will be displayed: on going (orange), completed ( green) , completed with X errors (red)

  
In case of errors:

- Total number of errors encountered.
- A brief description of each error.
-  Display only the last XY errors (this limit should be configurable in the application properties).

All synchronization information will be stored locally. The user will have the option to either clear all synchronization session data or selectively delete the data for a specific session.

**Comment by ameen4455** [comment]
Can I take up this issue?

**Comment by icrc-fdeniger** [comment]
Sure and thanks a lot @ameen4455

**Comment by parthfloyd** [comment]
Thanks @icrc-fdeniger 
I've merged PR referring to this issue. 

For clarity for this description "All synchronization information will be stored locally. The user will have the option to either clear all synchronization session data or selectively delete the data for a specific session." I've implemented considering to delete the "Synchronization session data" & not the (downloaded/uploaded) resources of that session.
